cmd/osbuild-worker/jobimpl-koji-finalize.go

```diff
@@ -150,7 +150,7 @@ func (impl *KojiFinalizeJobImpl) Run(job worker.Job) error {
 		})
 		images = append(images, koji.Image{
 			BuildRootID:  uint64(i),
```

Comment:
It is. From https://docs.pagure.org/koji/content_generator_metadata/#buildroots :

> id: An id for this buildroot entry. Only needs to be consistent within this file (it will be referenced by the output). Can be synthetic/generated/random.

Reply:
Thank you!